src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Logo from "../Logo";
+
+function Footer() {
+  return (
+    <section className="relative overflow-hidden py-10 bg-gray-400 border border-t-2 border-t-black">
+      <div className="relative z-10 mx-auto max-w-7xl px-4">
+        <div className="-m-6 flex flex-wrap">
+          <div className="w-full p-6 md:w-1/2 lg:w-5/12">
+            <div className="flex h-full flex-col justify-between">
+              <div className="mb-4 inline-flex items-center">
+                <Logo width="100px" />
+              </div>
+              <div>
+                <p className="text-sm text-gray-600">
+                  &copy; Copyright 2023. All Rights Reserved by DevUI.
```

Comment:
try to implement internationalisation from start. So that all of your strings also would be at a single place. Checkout out i8n

Reply:
will configure in next stage of development